src/components/types/Post.tsx

```diff
@@ -0,0 +1,11 @@
+import { User } from './User';
+import { Comment } from './Comment';
+
+export interface Post {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+  user: User | undefined,
+  comments: Comment[] | null,
```

Comment:
`comments` property will never be `null` because the `filter` method returns array in any case.

Reply:
Yes, @cyb1ove is correct.